pkg/driverbuilder/builder/flatcar.go

```diff
@@ -143,15 +144,6 @@ func fetchFlatcarPackageListURL(architecture kernelrelease.Architecture, flatcar
 	return urls
 }
 
-func fetchFlatcarKernelConfigURL(architecture kernelrelease.Architecture, flatcarChannel, flatcarVersion string) []string {
```

Comment:
We don't need to fetch kernel confgi URL anymore, because we expect kernelconfig to be already passed as `kernelconfigdata` option.